dashel/dashel-posix.cpp

```diff
@@ -563,35 +575,50 @@ namespace Dashel
 			}
 		}
 	};
-	
+
 	//! Poll a socket file descriptor for either a local Unix domain socket (tcppoll:sock=N) or a
 	//! remote TCP/IP socket (tcppoll:host=HOST;port=PORT). Delegates fd choice to getOrCreateSocket.
 	//! Poll streams are used to include sockets that will be read or written by client code in the
 	//! Dashel polling loop. Dashel itself neither reads from nor writes to the socket. PollStream will
 	//! call Hub::incomingData(stream) exactly once when its socket is polled with POLLIN in Hub::step.
-	class PollStream: public SelectableStream
+	class PollStream : public SelectableStream
 	{
 	public:
-		explicit PollStream(const std::string& targetName) :
-		Stream(targetName),
-		SelectableStream(targetName)
+		explicit PollStream(const std::string& targetName) : Stream(targetName), SelectableStream(targetName)
 		{
 			target.add("tcppoll:host;port;connectionPort=-1;sock=-1");
 			target.add(targetName.c_str());
 			fd = getOrCreateSocket(target);
-			dtorCloseSocket = target.get<int>("sock") < 0 && fd >= 0; // if getOrCreateSocket created the socket we will have to close it
+			dtorCloseSocket = target.get<int>("sock") < 0 &&
```

Comment:
This is less readable than the previous version, probably the best is to place the comment on its own line.